### PR TITLE
Longest Palindromic Substring Python

### PR DESCRIPTION
**Pull request by Muskan0**
#455

### CODE REVIEW

**Comment by Muskan0** [comment]
@tarun26091999 please merge it.

File: Code/Python/longest_palindromic_substring.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+'''
+for _ in range(int(input())):
+    s=input()
+    n=len(s)
+    # making a n * n matrix
+    t=[[0 for i in range(n)] for j in range(n)]
+
+    # in worst case, length of longest palindrome would be 1
+    ans=1
+    # start of the palindrome
```
Comment (rish-singhal) [inline Code/Python/longest_palindromic_substring.py:42]: Hey try out pep8 and solve the issues.

File: Code/Python/longest_palindromic_substring.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Output:
+aabbaa
+Test Case 2:
+2
+abcd
+zjjfjjwax
+Output:
+a
+jjfjj
+Time Complexity: O(n^2)
```
Comment (rish-singhal) [inline Code/Python/longest_palindromic_substring.py:92]: Describe what n is here

File: Algorithm/DP/readme.md [hunk 3]
```diff
@@ -47,6 +47,8 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 - Minimum Number of Deletions To Make a String Palindrome ---->[C++](/Code/C++/min_deletions_to_make_string_palindrome.cpp)
 
+- Longest Palindromic Substring ----> [Python](/Code/Python/longest_palindromic_substring.py)
+
 - Climbing Stairs ----> [C++](/Code/C++/Climbing_Stairs.cpp)
```
Comment (rish-singhal) [inline Algorithm/DP/readme.md:50]: Sort this whole list.

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Comment by Muskan0** [comment]
@rish-singhal Please review it.

File: Algorithm/DP/readme.md [hunk 4]
```diff
@@ -51,6 +51,8 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 - Word Wrap Problem ----> [C++](/Code/C++/word_wrap.cpp)
 
-- Painting Fence Algorithm ----> [Python](/Code/Python/PaintFenceAlgo.py) | [Java](/Code/Java/Paint_fence_algo.java) 
+- Painting Fence Algorithm ----> [Python](/Code/Python/PaintFenceAlgo.py) | [Java](/Code/Java/Paint_fence_algo.java)
+
+- Longest Palindromic Substring ----> [Python](/Code/Python/longest_palindromic_substring.py)
```
Comment (rish-singhal) [inline Algorithm/DP/readme.md:56]: Sort this whole list

**Comment by rish-singhal** [comment]
Why have you changed the readme in DP section? It should be changed in strings

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Comment by Muskan0** [comment]
> Why have you changed the readme in DP section? It should be changed in strings

I used the DP approach, that's why I entered it in DP section. I changed it now.

File: string/readme.md [hunk 5]
```diff
@@ -31,3 +31,4 @@ A string is a class that contains a char array, but automatically manages it for
 * Maximum occuring character in a string ---->[C++](/Code/C++/max_count.cpp)
 * Checking for Anagram of two string ---->[Java](/Code/Java/Anagram.java)
 * Kth Minimum in array ---->[Python](/Code/Python/Kth_Minimum_in_array.py)
```
Comment (rish-singhal) [inline string/readme.md:33]: Sort both the readme files,

File: Algorithm/DP/readme.md [hunk 6]
```diff
@@ -51,6 +51,4 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 - Word Wrap Problem ----> [C++](/Code/C++/word_wrap.cpp)
```
Comment (rish-singhal) [inline Algorithm/DP/readme.md:52]: Sort this readme too

**Review by rish-singhal — CHANGES_REQUESTED** [review]
You just need to sort both readme.

File: string/readme.md (hunk 5 above)
Comment (rish-singhal) [reply string/readme.md:33]: This list is not sorted still

File: Code/Python/longest_palindromic_substring.py [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+            c  b  b  d
+          c [[1, 0, 0, 0],
+          b [0, 1, 1, 0],
+          b [0, 0, 1, 0],
+          d [0, 0, 0, 1]]
+          This example only has 2 length substring length.
+        5. Make a count of the maximum length of palindromic substring found
+           and it's start index throughout the filling of the table.
+'''
+for _ in range(int(input())):
```
Comment (rish-singhal) [inline Code/Python/longest_palindromic_substring.py:44]: Add empty line here

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Algorithm/DP/readme.md [hunk 8]
```diff
@@ -35,22 +35,20 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 ## Questions :
 
-- Knapsack ----> [C++](/Code/C++/knapsack.cpp)
+- Climbing Stairs ----> [C++](/Code/C++/Climbing_Stairs.cpp)
 
 - Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp)
```
Comment (rish-singhal) [inline Algorithm/DP/readme.md:41]: remove spaces in between

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Pull request merged**